raiden/network/transport/matrix/rtc/web_rtc.py

```diff
@@ -620,15 +603,17 @@ def stop(self) -> None:
         self._reset_state()
         self._send_hangup_messages()
```

Comment:
the second line will be a noop because `reset_state()` will empty the dictionary

Reply:
Indeed, you're right. What was the purpose of that before (my changes didn't touch that part)? It seems like those hangup messages were never sent. Shall I just remove that method?

Reply:
sounds reasonable. I think they should be sent as this is called in the stop method to notify the partners that the node is currently shutting down the channels
(In the past we had a failure in state showing that the datachannel was open although the partner already closed it)

Reply:
Maybe calling the hangup method before closing the channels makes sense

Reply:
> Maybe calling the hangup method before closing the channels makes sense

Yes, I'll do that and see if it goes well.